src/Compilers/CSharp/Portable/Binder/Binder.QueryUnboundLambdaState.cs

```diff
@@ -50,7 +50,7 @@ public override bool HasExplicitReturnType(out RefKind refKind, out TypeWithAnno
             public override RefKind RefKind(int index) { return Microsoft.CodeAnalysis.RefKind.None; }
             public override ScopedKind DeclaredScope(int index) => ScopedKind.None;
             public override MessageID MessageID { get { return MessageID.IDS_FeatureQueryExpression; } } // TODO: what is the correct ID here?
-            public override Location ParameterLocation(int index) { return _parameters[index].Locations[0]; }
+            public override Location ParameterLocation(int index) { return _parameters[index].GetFirstLocation(); }
```

Comment:
this commit should be the mechanical change of Locations[0] to GetFirstLocation()